### PR TITLE
Assignment Report Fixes

### PR DESCRIPTION
**Pull request by soachishti**
1. Added "username"
2. Added "Submission date"
3. Table columns are sortable now
4. Score instead of percent
5. Code highlight

### CODE REVIEW

File: AutoGrade/admin.py [hunk 1]
```diff
@@ -67,8 +67,6 @@ def __init__(self, *args, **kwargs):
             instructor = Instructor.objects.filter(user=self.current_user).first()
             self.fields['course'].queryset = Course.objects.filter(instructor=instructor)
 
-    list_filter = ('course', )
```
Comment (recluze) [inline AutoGrade/admin.py:70]: Er ... I think this got killed in this commit. Need this in the file.

**Review by recluze — DISMISSED** [review]
Keep the admin file and commit the rest. Let's also create a `release` branch so that we can keep prod separate from dev. 

I will test tonight inshaallah in detail and try to deploy it on the server tomorrow. (Can't do it now since the machines are off =[ ... )

File: AutoGrade/admin.py (hunk 1 above)
Comment (soachishti) [reply AutoGrade/admin.py:70]: `list_filter` attributes are placed in `AssignmentFormAdmin(forms.ModelForm)` and `AssignmentModelAdmin(admin.ModelAdmin)` but `list_filter` has no meaning in `forms.ModelForm` instance. 

@recluze

Comment (recluze) [reply AutoGrade/admin.py:70]: Oh, ok. Sorry about that.

**Pull request merged**